### ISSUE TITLE
support dumping AST created by typed_ast

### ISSUE THREAD

**Issue by abulka** [asottile/astpretty#12]
It would be good if astpretty supported AST's built with Guido's [typed_ast](https://github.com/python/typed_ast) which is a Python 3 package which parses both Python 2 and Python 3 and produce the standard Python AST.

There is a hack I came up with which works, but perhaps there should be a more elegant official way in the future.

```
# import typed_ast.ast27 as ast
import typed_ast.ast3 as ast

import astpretty
astpretty.ast = ast  # monkey patch the ast away from built in python to use typed_ast's instead
```

**Comment by asottile** [comment]
I like this idea.  Going to take a stab at it!

**Comment by asottile** [comment]
Thanks again for the issue!  This is available in v1.4.0! :tada:
